java/tv/sage/ModuleGroup.java

```diff
@@ -293,27 +293,32 @@ else if (cryptoKeyObj instanceof Object[])
         }
       }
       file.createNewFile();
-      sage.FastRandomFile widgetDBout = new sage.FasterRandomFile(file, cryptoKey == null ? "rwd" : "rwdc", sage.Sage.I18N_CHARSET);
-      if (cryptoKey != null) widgetDBout.setCrypto(cryptoKey);
+      sage.io.SageDataFile widgetDBout;
+
+      if (cryptoKey == null)
```

Comment:
I saw that the only thing using it was the method that basically passes null for the encryption key. Per your suggestion, it's now gone. I did occasionally run into your comments about paranoia in the code; at times I get a good laugh. :)